src/sentry/tasks/groupowner.py

```diff
@@ -99,11 +103,13 @@ def _process_suspect_commits(
                                     "user_id": owner_id,
                                     "project_id": project.id,
                                     "organization_id": project.organization_id,
+                                    "context__asjsonb__commitId": owner_commits[owner_id],
```

Comment:
do we have any tests which will test that this task will successfully update a group owner with the right fields?